src/code-components/RichTextArea/Editor.tsx

```diff
@@ -55,7 +56,11 @@ const Editor = forwardRef<Quill | null, EditorProps>(
         const editorHtml = quill.root.innerHTML.trim();
 
         if (value?.trim() !== editorHtml) {
-          const contents = quill.clipboard.convert({ html: value ?? "" });
+          const dirty = value ?? "";
+          const sanitizedHtml = sanitizeHtml(dirty, {
+            allowedTags: sanitizeHtml.defaults.allowedTags.concat(["img"]),
```

Comment:
Is it exactly the same sanitization mechanism as we have on back-end, or did you choose a different one? If a different one - why?